server/gokbg3/grails-app/i18n/messages_en.properties

```diff
@@ -377,7 +377,9 @@ identifier.value.notNull=The Identifier must have a value!
 identifier.value.notUnique=An Identifier with this value already exists!
 identifier.value.noNamespace=No Namespace configured!
 identifier.create.error=Could not create Identifier!
-identifier.link.unique=This identifier is already connected to this component!
+identifier.no.unique.by.component=This identifier is already exists for this component!
```

Comment:
already exists or is already connected to

Reply:
is already exists